branchpro/tests/test_incidence_number_plot.py

```diff
@@ -0,0 +1,89 @@
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
+import unittest
+from unittest.mock import patch
+
+import numpy as np
+import numpy.testing as npt
+import pandas as pd
+
+import branchpro as bp
+
+
+class TestIncidenceNumberPlotClass(unittest.TestCase):
+    """
+    Test the 'IncidenceNumberPlot' class.
+    """
+    def test__init__(self):
+        bp.IncidenceNumberPlot()
+
+    def test_add_data(self):
+        df = pd.DataFrame({
+            "Time": [1, 2, 3, 5, 6],
```

Comment:
Use single quotes for the strings, it has no effect but pep8 does say to be consistent in your project